multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -36,14 +36,31 @@ coreos:
       command: start
       enable: true
       content: |
+{{ if isset .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=load-rkt-stage1.service
+        After=load-rkt-stage1.service
+{{ end }}
         [Service]
         Environment=KUBELET_VERSION={{.K8sVer}}
         Environment=KUBELET_ACI={{.HyperkubeImageRepo}}
-        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf --mount volume=dns,target=/etc/resolv.conf"
+        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf \
+          --mount volume=dns,target=/etc/resolv.conf \
+          --volume rktbin,kind=host,source=/usr/bin/rkt \
+          --mount volume=rktbin,target=/usr/bin/rkt \
```

Comment:
A write-api also solves this issue in the longer term, and this is kinda a solution in the same spirit (in that it's just shoving the problem off as "run rkt on the host, not in my app")